File: Classes/DozukiInfoViewController.m
//
//  DozukiInfoViewController.m
//  iFixit
//
//  Created by David Patierno on 8/16/11.
//  Copyright (c) 2011 iFixit. All rights reserved.
//

#import "iFixit-Swift.h"
#import "DozukiInfoViewController.h"

@implementation DozukiInfoViewController

@synthesize dssvc;

- (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil
{
    self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil];
    if (self) {
        // Custom initialization
        self.dssvc = [[DozukiSelectSiteViewController alloc] initWithSimple:YES];
        self.title = NSLocalizedString(@"Back", nil);
    }
    return self;
}

- (void)showList {
    [self.navigationController pushViewController:dssvc animated:NO];
}

#pragma mark - View lifecycle

- (void)viewDidLoad {
    [super viewDidLoad];
    
    [self willRotateToInterfaceOrientation:self.interfaceOrientation duration:0];
}


@end

File: Classes/DozukiInfoViewController.swift
//
//  DozukiInfoViewController.m
//  iFixit
//
//  Created by David Patierno on 8/16/11.
//  Copyright (c) 2011 iFixit. All rights reserved.
//

class DozukiInfoViewController : UIViewController {

    var dssvc: DozukiSelectSiteViewController!

    override init(nibName:String?, bundle:NSBundle?) {
        
        super.init(nibName:nibName, bundle:bundle)

        // Custom initialization
        self.dssvc = DozukiSelectSiteViewController(simple:true)
        self.title = NSLocalizedString("Back", comment:"")
    }

    required init?(coder aDecoder: NSCoder) {
        super.init(coder:aDecoder)
    }

    func showList() {
        self.navigationController?.pushViewController(dssvc, animated:false)
    }

    // MARK: - View lifecycle

    override func viewDidLoad() {
        super.viewDidLoad()
    
        willRotateToInterfaceOrientation(self.interfaceOrientation, duration:0)
    }

}
